File: 1033-moving-stones-until-consecutive/1033-moving-stones-until-consecutive.cpp
class Solution {
public:
    vector<int> numMovesStones(int a, int b, int c) {
        vector<int>temp={a,b,c};
        sort(temp.begin(),temp.end());
        if(temp[2]-temp[0]==2)return {0,0};
        return {min(temp[1]-temp[0],temp[2]-temp[1])<=2?1:2,temp[2]-temp[0]-2};
    }
};

File: 1033-moving-stones-until-consecutive/1033-moving-stones-until-consecutive.java
class Solution {
    public int[] numMovesStones(int a, int b, int c) {
        int[]arr={a,b,c};
        Arrays.sort(arr);
        if(arr[2]-arr[0]==2)return new int[] {0,0};
        return new int[]{Math.min(arr[2]-arr[1],arr[1]-arr[0])<=2?1:2,arr[2]-arr[0]-2};
    }
}